packages/itwin/measure-tools/src/api/Measurement.ts

```diff
@@ -59,26 +60,46 @@ export namespace DrawingMetadata {
       return undefined;
     const origin = obj.origin?.toJSONXY();
     const extents = obj.extents?.toJSONXY();
-    const masterOrigin = obj.sheetToWorldTransform?.masterOrigin.toJSONXYZ();
-    const sheetTov8Drawing = obj.sheetToWorldTransform?.sheetTov8Drawing.toJSON();
-    const v8DrawingToDesign = obj.sheetToWorldTransform?.v8DrawingToDesign.toJSON();
-    if (origin !== undefined)
-      return { origin, extents, worldScale: obj.worldScale, drawingId: obj.drawingId, sheetToWorldTransform: (masterOrigin !== undefined && sheetTov8Drawing !== undefined && v8DrawingToDesign !== undefined) ? {masterOrigin, sheetTov8Drawing, v8DrawingToDesign}: undefined };
+    if (origin !== undefined) {
+      return {
+        origin, extents, drawingId: obj.drawingId,
+        sheetToWorldTransformProps: {
+          DVDOrigin: obj.sheetToWorldTransformProps?.DVDOrigin,
```

Comment:
There's probably a way to spread the values instead of doing direct assignment, something like 
`sheetToWorldTransformProps: obj.sheetToWorldTransformProps ? { ...obj.sheetToWorldTransformProps } : undefined`.

Same with the code in `fromJSON`